### DISCUSSION TITLE
Force Sync for Piped?

### DISCUSSION THREAD

**Discussion by NikunjK8** [fast4x/RiMusic#2413] [Q&A]
I logged in with same piped account in rimusic and harmony music desktop version. However only three of my albums and some last played songs appeared in Harmony music. One thing i noticed is my network speed doesn't go up in Rimusic and stays on 0kbps after logging in. Which means Rimusic isn't syncing properly. Is there any way to force sync my data?

**Comment by fast4x** [comment]
Can add here link to your piped playlists?

**Reply by NikunjK8** [reply]
Sorry i don't understand, if you're talking about playlists on my piped account then i don't have any as i don't use Internet playlists.
I just want my bookmarked albums, followed artists, local playlists and favourited songs to sync.

**Reply by fast4x** [reply]
But all of these sync you want to do with piped?
